README.pt-br.md

```diff
@@ -33,18 +33,32 @@ Em seguida execute:
 ruby minha_app.rb
 ```
 
-Acesse: [http://localhost:4567](http://localhost:4567)
+Acesse em: [http://localhost:4567](http://localhost:4567)
+<<<<<<< HEAD
+=======
 
-É recomendado também executar `gem install thin`. Caso esta gem esteja disponível, o
-Sinatra irá utilizá-la.
+Códigos alterados só terão efeito após você reiniciar o servidor.  
+Por favor, reinicie o servidor após qualquer mudança ou use 
+[sinatra/reloader](http://www.sinatrarb.com/contrib/reloader).
+>>>>>>> c4b7e1e1... intro update
+
+<<<<<<< HEAD
+Códigos alterados só terão efeito após você reiniciar o servidor.  
+Por favor, reinicie o servidor após qualquer mudança ou use 
+[sinatra/reloader](http://www.sinatrarb.com/contrib/reloader).
+
+=======
+>>>>>>> b6f0256c... readme translation updates
+É recomendado também executar `gem install thin`. Caso esta gem esteja 
+disponível, o Sinatra irá utilizá-la.
```

Comment:
I think you're something wrong.. Please do the correct fix for conflicting.

Reply:
Can you check if everything is alright now?

Reply:
I'll try to fix this now.